examples/cpp/dds/BasicConfigurationExample/BasicConfigurationSubscriber.cpp

```diff
@@ -182,6 +184,8 @@ bool HelloWorldSubscriber::init(
     if (transient)
     {
         rqos.durability().kind = TRANSIENT_LOCAL_DURABILITY_QOS;
+        // mimic publisher behaviour. Assure the user see all samples are received
+        rqos.history().kind = KEEP_ALL_HISTORY_QOS;
```

Comment:
If set with both reliability and durability, please add it also to Publisher

Reply:
done